package.json

```diff
@@ -8,15 +8,19 @@
   "bin": {
     "station": "./bin/station.js"
   },
+  "engines": {
+    "node": ">=18.12"
```

Comment:
Hmm, I see that the CI is running on Node.js 16 too. Is there any particular reason why we should support that version line?

Either way, we should use the same minimal supported Node.js version in all three places - CI workflows, package.json, and documentation.

Reply:
Node 16 has EOL 2023-09-11, so is still a relevant and supported version (https://github.com/nodejs/Release). Or would you like to choose different criteria?

Reply:
Are you ok with supporting node 16? If yes, please adjust the other places to match

Reply:
The fewer versions we support, the less maintenance work for us.

Considering that this is a CLI, not a library, I would assume people have more flexibility in choosing the Node.js version and thus be able to run on the latest LTS.

Anyhow, the goal of this PR is to add docs, so I'll update my changes to include support for Node.js 16.x.

Reply:
Resolved in https://github.com/filecoin-station/core/pull/47/commits/5cb8a745a1c020e995bfffc2b7c12acc118f806b